docker-compose.dev.yaml

```diff
@@ -0,0 +1,34 @@
+version: '3.9'
```

Comment:
what about making a `/dev/` subfolder and put everything dev related in it (ex: this yaml file) instead of the root of the repo?